back/app/controllers/web_api/v1/activities_controller.rb

```diff
@@ -1,37 +1,31 @@
 # frozen_string_literal: true
 
 class WebApi::V1::ActivitiesController < ApplicationController
-  before_action :set_post_type_and_id
-  skip_after_action :verify_policy_scoped
   skip_before_action :authenticate_user, only: %i[index]
 
   def index
-    activities = ActivitiesFinder.new(
-      finder_params,
-      includes: :user,
-      scope: policy_scope(Activity)
-    ).find_records
-    paginated_activities = paginate activities
+    @activities = policy_scope Activity
```

Comment:
We use global `@` variables in a lot of our controllers - but I don't see that it should be necessary as the variable is not used outside of this method